internal/resourcestatus/resourcekey.go

```diff
@@ -5,21 +5,29 @@ import (
 	"sort"
 	"strings"
 
+	"github.com/rancher/fleet/internal/cmd/controller/summary"
 	fleet "github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
 )
 
-func SetResources(list *fleet.BundleDeploymentList, status *fleet.StatusBase) {
+func setBaseStatusResources(list *fleet.BundleDeploymentList, status *fleet.StatusBase) {
 	s := summaryState(status.Summary)
 	r, errors := fromResources(list, s)
 	status.ResourceErrors = errors
-	status.ResourceCounts = countResources(r)
 	status.Resources = merge(r)
+	status.ResourceCounts = sumResourceCounts(list)
+}
+
+func SetGitRepoResources(list *fleet.BundleDeploymentList, status *fleet.GitRepoStatus) {
+	setBaseStatusResources(list, &status.StatusBase)
+	status.PerClusterResourceCounts = resourceCountsPerCluster(list)
+}
+
+func SetHelmAppResources(list *fleet.BundleDeploymentList, status *fleet.HelmAppStatus) {
+	setBaseStatusResources(list, &status.StatusBase)
```

Comment:
My understanding of this is that we don't yet want/need per-cluster resource counts on HelmApp because that resource is not yet visible in the Rancher UI. Is that correct, or am I missing an important aspect of this?